index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Pat's Salmon Cookies</title>
+    <nav></nav>
+    <link type="text/css" rel="stylesheet" href="stylesheet.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  </head>
+  <body>
+    <main>
+      <section>
+        <article class="top"><h1>Pat's Salmon Cookies</h1>
+        </article>
```

Comment:
If you're gonna put the contents on one line with the opening tag, put the closing tag on the same line. Otherwise indent the contents on a new line inside the opening and closing tags.